dev/com.ibm.ws.webcontainer.security_test.servlets/src/com/ibm/ws/webcontainer/security/test/servlets/ServletClientImpl.java

```diff
@@ -580,6 +580,7 @@ protected void setSSOCookieForLastLogin(HttpMessage httpMessage) {
     protected String getSSOCookie(HttpMessage httpMessage, String cookieName) {
         logger.info("getSSOCookie");
         String cookieValue = null;
+        String cookiePath = null;
```

Comment:
same comment as below

If the local cookiePath value is never set to anything besides null at initialization, then we can just use null directly, and remove the local String cookiePath.